__test__/api/comment-vote.test.js

```diff
@@ -7,71 +7,64 @@ import { describeIfEndpoint } from "../helpers/conditionalTests";
 
 describeIfEndpoint(
   "PATCH",
-  "/comments/:commentId/votes",
-  "PATCH '/comments/:commentId/votes'",
+  "/comments/:commentId/votes/upvote",
+  "PATCH '/comments/:commentId/votes/upvote'",
   () => {
     test("Should upvote a comment", async () => {
-      const comment = new CommentModel({
+      const commentObject = {
         content: "this is a comment",
         ownerId: "useremail@email.com",
         origin: "123123",
         applicationId: mongoose.Types.ObjectId(),
-      });
+      };
+      const comment = new CommentModel(commentObject);
       await comment.save();
 
-      const res = await request.patch(`/comments/${comment._id}/votes`).send({
-        voteType: "upvote",
-      });
+      const res = await request
+        .patch(`/comments/${comment._id}/votes/upvote`)
+        .send({
+          ownerId: commentObject.ownerId,
+        });
       if (res.status === 404) {
```

Comment:
the utility function `describeIfEndpoint` handles 404 error, so you can completely remove the `If` block